<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>extr</class>
 <widget class="QGroupBox" name="extr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>106</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="title">
   <string>extrinsic parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,0,0,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label_58">
       <property name="toolTip">
        <string>Translate the coordinate system in x-direction.</string>
       </property>
       <property name="text">
        <string>rotation:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_59">
       <property name="toolTip">
        <string>Translate the coordinate system in x-direction.</string>
       </property>
       <property name="text">
        <string>translation:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="PDoubleSpinBox" name="trans1">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PDoubleSpinBox" name="rot1">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="PDoubleSpinBox" name="trans2">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="PDoubleSpinBox" name="trans3">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>-500.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="PDoubleSpinBox" name="rot3">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="PDoubleSpinBox" name="rot2">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_62">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>3D/2D Points:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="coordLoad3DCalibPoints">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Load 3D points to corresponding 2D image points</string>
       </property>
       <property name="text">
        <string>load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extrCalibFetch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Fetch marked 2D Points to loaded 3D points</string>
       </property>
       <property name="text">
        <string>fetch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extrCalibSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Save 3D and/or 2D points to extrinisc calib file</string>
       </property>
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extrCalibShowPoints">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Show the saved 2D/3D point correspondences in the file</string>
       </property>
       <property name="text">
        <string>show</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extrCalibShowError">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Show the reprojection error of extrinsic calibration</string>
       </property>
       <property name="text">
        <string>error</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>pdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rot1</tabstop>
  <tabstop>rot2</tabstop>
  <tabstop>rot3</tabstop>
  <tabstop>trans1</tabstop>
  <tabstop>trans2</tabstop>
  <tabstop>trans3</tabstop>
  <tabstop>coordLoad3DCalibPoints</tabstop>
  <tabstop>extrCalibFetch</tabstop>
  <tabstop>extrCalibSave</tabstop>
  <tabstop>extrCalibShowPoints</tabstop>
  <tabstop>extrCalibShowError</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
